tests/Hooks/Migration/Unit/ConfiguresMigrationsTest.php

```diff
@@ -29,21 +29,45 @@ class ConfiguresMigrationsTest extends ConfigureHookTestCase
 
     protected $eventClass = ConfigureMigrations::class;
 
+    protected function afterSetUp()
+    {
+        resolve('migrator')->path(__DIR__);
+        parent::afterSetUp();
+    }
+
     /**
      * @dataProvider tenantEventsProvider
      *
      * @test */
     public function it_can_add_paths_to_the_migrator($tenantEvent)
     {
         $this->events->listen($this->eventClass, function ($event) {
-            $event->path(__DIR__);
+            $event->path(realpath(__DIR__.'/..'));
         });
 
         $this->hook->for(new $tenantEvent($this->mockTenant()));
 
         $this->assertContains(
+            realpath(__DIR__.'/..'),
+            $this->hook->paths
+        );
+    }
+
+    /**
+     * @dataProvider tenantEventsProvider
+     *
+     * @test */
+    public function it_can_clear_paths_from_the_migrator($tenantEvent)
+    {
+        $this->events->listen($this->eventClass, function ($event) {
```

Comment:
First, assert that the path is still in the migrator there.

Reply:
👍 Good call.  Done